Assignment1_q2_Singly_Linked_List.py

```diff
@@ -0,0 +1,57 @@
+from Assignment1_q2_Node import *
+
+# My implementation of Singly Linked List
+class SinglyLinkedList(object):
+
+    def __init__(self, head = None):
+        self.head = head
+
+    # Inserting elements to the front of the list
+    def insert_first(self, data):
+        new_node = Node(data)
+        new_node.set_next_node(self.head)
+        self.head = new_node
+
+    # Get kth to the last element of the list - assignment question
```

Comment:
The function actually returns a `Node` rather than the element of the list.
From the user's perspective, they have a list of, e.g., strings and the want the k-th to last string, note the `Node`.